template/workflows/azure-kubernetes-service-kompose.yml

```diff
@@ -35,15 +35,15 @@ name: Build and deploy an app to AKS with Kompose
 
 on:
   push:
-    branches: [$default-branch]
+    branches: [{{BRANCHNAME}}]
```

Comment:
I believe this file should live in its own `kompose` subdir? @davidgamero to check me on that

Reply:
Yes, but I couldn't find any existing code for kompose, so I left it alone.

Reply:
Deleted the file since it is not used.

Reply:
yep if we were going to keep it @bfoley13 is correct, but since we don't use it that's also fine